### DISCUSSION TITLE
Welcome to Hexo Aurora Theme discussion! 🏳️‍🌈

### DISCUSSION THREAD

**Discussion by bennyxguo** [auroral-ui/hexo-theme-aurora#43] [General]
<img width="944" alt="image" src="https://user-images.githubusercontent.com/24708758/114228502-196c9000-99a9-11eb-87c7-de981ddc4872.png">

# Welcome to Hexo Aurora Theme's discussion

> If you are looking at this, this means you love this theme like I do!

## What can I do here?

- Ask question about the theme.
- Request help on the problem you are facing using the theme.
- A feature you really want to have, suggest it in the discussion.
- Make friends and share your thoughts.

## What do I need to beware of?

- No fighting.
- No intoxicating. (No trash talk)
- Be helpful and be nice to everyone else.
- There are not stupid questions, if you got a problem simply ask away.

**Comment by xiaoyu-666** [comment]
我觉得可以将这个主题用在vuePress上是个不错的主意

**Reply by bennyxguo** [reply]
我觉得后面是可以移植到 Vue Press 上的。

**Reply by xiaoyu-666** [reply]
那大概什么时候移植到vuePress上？？？

**Reply by xiaoyu-666** [reply]
因为我之前用vuePress开发过公司的技术文档网站，我对vuePress很熟悉，所以希望尽快将Aurora主题移植到vuePress上，感谢！！

**Reply by yuanliyuan-zhongyuan** [reply]
这个主题有群吗？我太难了
